debezium-core/src/test/java/io/debezium/transforms/outbox/EventRouterTest.java

```diff
@@ -80,6 +87,122 @@ public void canSkipDeletion() {
         assertThat(eventRouted).isNull();
     }
 
+    @Test
+    public void canSkipMessagesWithoutDebeziumCDCEnvelopeDueToMissingSchemaName() {
```

Comment:
Two minor nits:

* can you add the `@FixFor` annotation
* s/CDC/Cdc/ (i.e. abbreviations in identifiers should not be given all-uppercase, but just with the first letter)

Reply:
Done!